cmake/fetch_libtorch.cmake

```diff
@@ -1,17 +1,19 @@
-cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.28.6 FATAL_ERROR)
 
 include(FetchContent)
 
-set(CUDA_V "none" CACHE STRING "Determines libtorch CUDA version to download (11.8, 12.1 or none).")
+set(CUDA_V "none" CACHE STRING "Determines libtorch CUDA version to download (11.8, 12.1, 12.6 or none).")
```

Comment:
Looks like the currently supported version are 11.8, **12.4** and 12.6.

Reply:
Thanks, fixed!